index.html

```diff
@@ -1288,6 +1286,137 @@ <h2>DID Subject</h2>
 property. The value of <code>id</code> in the resolved <a>DID document</a> is
 expected to match the <a>DID</a> that was resolved.
       </p>
+        </section>
+
+        <section>
+        <h2>sameSubject</h2>
+        <p>
+        A <a>DID controller</a> MAY assert that two or more <a>DIDs</a> identify
+        the same <a>DID subject</a> using the <code>sameSubject</code> property. This
+        property is OPTIONAL.
+        </p>
+
+        <dl>
+        <dt>sameSubject</dt>
+        <dd>
+        The value of <code>sameSubject</code> MUST be a
+        <a data-cite="INFRA#lists">list</a> where each item in the list is a
+        <a>DID</a> that conforms to the rules in Section <a href="#did-syntax"></a>.
+        </dd>
+      </dl>
+        <p>
+        From a semantic standpoint, the <a>DID controller</a> is asserting that
+        each <a>DID</a> in the list identifies the same <a>DID subject</a> as
+        the <a>DID</a> described by the <a>DID document</a> (i.e., the value of
+        the <code>id</code> property).
+        </p>
+        <p class="note">
+        WARNING: A <code>sameSubject</code> assertion that two or more <a>DIDs</a>
+        identify the same <a>DID subject</a> does not prove this assertion is
+        true. A malicious <a>DID controller</a> may attempt to impersonate a
+        <a>DID subject</a> by making a false <code>sameSubject</code> assertion.
+        Therefore it is STRONGLY RECOMMENDED that a requesting party obtain
+        independent verification of a <code>sameSubject</code> assertion. One
+        method for doing this is to resolve the <code>sameSubject</code> <a>DID</a>
+        to its own <a>DID document</a> and then verify that it contains the
+        inverse <code>sameSubject</code> assertion (i.e., a backpointer). See
+        the <a href="#security-considerations"></a> and
+        <a href="#privacy-considerations"></a> sections for more information.
+        </p>
+        <p class="note">
+        A <code>sameSubject</code> assertion is not the same as an
+        <code>owl:sameAs</code> assertion. In other words, it does not assert
+        that two or more RDF graphs represented as <a>DID documents</a> can be
+        merged. It only asserts that two or more <a>DIDs</a> identify the same
+        <a>DID subject</a>.
+        </p>
+        </section>
+
+      <section>
+        <h2>representation</h2>
```

Comment:
This property is begging to be abused from a GDPR perspective. :)

There are better and more specific properties in schema.org... `image`, `url`, etc. We shouldn't advise implementers to express this sort of information in the DID Document... it should be done externally.

Please split the `representation` PR into a separate PR... as the `sameSubject` PR is far less controversial. I'm concerned that this PR is going to be held up because of the `representation` PR... they should progress on different timelines.